client/components/jetpack-logo/index.jsx

```diff
@@ -3,11 +3,47 @@
  */
 import React from 'react';
 
-const JetpackLogo = ( { size } ) => {
-	const svgPath = 'M42,0A42,42,0,1,0,84,42,42,42,0,0,0,42,0ZM40.36,' +
+const JetpackLogo = ( { size, full } ) => {
+	const svgIconPath = 'M42,0A42,42,0,1,0,84,42,42,42,0,0,0,42,0ZM40.36,' +
 		'51.81,27.17,48.45a5.21,5.21,0,0,1-3.23-7.65L40.36,' +
 		'12.35Zm20.32-8.6L44.27,71.65V32.19l13.19,3.36A5.21,5.21,0,0,1,60.69,43.2Z';
 
+	if ( full === true ) {
+		return (
+			<svg height={ size } className="jetpack-logo" viewBox="0 0 183 32">
```

Comment:
The other svg includes both `width` and `height`. I checked in recent chrome, firefox and safari and this seems to work fine, however I recall some browsers doing wonky things when size isn't well defined. Should we add `width={ size * 183 / 32 }`?

Is there anything lost if we run this through a minifier like `svgo`?

A quick run gives the following, which is a nice single path:

```jsx
<svg height={ size } class="jetpack-logo" viewBox="0 0 183 32">
    <title>
        Jetpack
    </title>
    <path d="M50 7h-5.968L49 11.874v6.14c0 1.091-.217 2.595-1.016 3.049-.749.426-1.715.307-2.953-.328C43.736 20.07 43.213 19.217 43 19l-2 4c.111.1 2.562 2.669 6 2.669 2.182 0 4.212-.563 5.469-1.919C53.527 22.625 54 20.559 54 18.311v-7.416C54 8.727 52.169 7 50 7zm31 3.946h5V25h4.987V10.946H96V7.011H81zm28.804-3.935H102V25h4.931v-5h2.642c2.149 0 3.902-.529 5.125-1.62 1.224-1.091 2.233-2.829 2.233-4.88 0-1.818-.877-3.513-1.969-4.637-1.19-1.19-2.876-1.852-5.158-1.852zm1.356 8.499c-.562.364-1.29.496-1.951.496h-2.278v-5.192h2.344c.694 0 1.554.099 2.116.628.463.43 1.04 1.058 1.04 1.918 0 1.058-.643 1.786-1.271 2.15zm18.771-9.014c-2.48 0-4.594 1.176-5.85 2.432-1.422 1.422-2.116 3.505-2.116 5.39V25h4.967v-5h6v5h5V14.319c0-1.885-.694-3.968-2.116-5.39-1.258-1.256-3.339-2.433-5.885-2.433zm3 8.082V16h-6v-1.422c0-1.125-.009-1.949.52-2.578.628-.694 1.554-1.025 2.48-1.025.992 0 1.885.298 2.546 1.025.529.629.454 1.453.454 2.578zM61.079 24.989H75V21.12h-9v-3.24h7v-3.87h-7v-3.108h9V7H61.079zm96.487-5.019a18.55 18.55 0 0 1-.282.143l-.05.024c-1.007.481-2.119.83-3.381.83-1.521 0-2.877-.529-3.77-1.455-.959-.893-1.521-2.215-1.521-3.802 0-1.323.463-2.48 1.224-3.406.926-1.125 2.348-1.819 4.133-1.819 1.021 0 1.828.18 2.682.524l.156.065.278.121.119.055.186.09c.203.1.415.21.638.338l1.715-3.603a7.478 7.478 0 0 0-1.115-.61c-1.265-.59-2.762-.982-4.858-.982-2.844 0-5.489 1.224-7.275 3.141-1.521 1.62-2.447 3.704-2.447 6.085 0 2.877 1.058 5.192 2.778 6.812 1.719 1.62 4.134 2.48 6.878 2.48 2.308 0 4.002-.531 5.442-1.258l.08-.04.017-.009c.236-.119.49-.254.689-.379l-1.778-3.642c-.188.11-.368.208-.538.297zM182 7.011h-5.793L171 12.729V7h-3v.011h-2V25h5v-6.459l.489-.518L176.758 25H182l-7.535-10.086zM16 0C7.163 0 0 7.163 0 16s7.163 16 16 16 16-7.163 16-16S24.837 0 16 0zm-1 19.736l-5.026-1.279a1.985 1.985 0 0 1-1.23-2.916L15 4.704v15.032zm8.255-3.277L17 27.296V12.264l5.026 1.279a1.985 1.985 0 0 1 1.229 2.916z"/>
</svg>
```

Reply:
Feel free to optimize it and fix it. I don't think anything is lost if it still renders correctly. I guess we'd lose the ability to target specific shapes with CSS for animations and such, but I don't know that we need that for this SVG.